__tests__/app.test.js

```diff
@@ -243,11 +243,120 @@ describe("/api/articles", () => {
       });
     });
     describe("POST", () => {
-      //   test("POST 400: returns 400 when no object is sent", () => {
-      //     return request(app)
-      //       .post("/api/articles/:article_id/comments")
-      //       .expect(400);
-      //   });
+      test("POST 201: returns 201 and the inserted comment object when sent a valid comment object", () => {
+        const article_id = 1;
+        const newComment = {
+          body: "comment body",
+          username: "lurker",
+        };
+        return request(app)
+          .post(`/api/articles/${article_id}/comments`)
+          .send(newComment)
+          .expect(201)
+          .then(({ body: { comment } }) => {
+            expect(comment.article_id).toBe(article_id);
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
```

Comment:
While you are checking this on line 257, you aren't expecting any number for the article_id.